modules/config-htpasswd-idp.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * assemblies/config-identity-providers.adoc
+
+[id="config-htpasswd-idp_{context}"]
+= Configuring an HTPasswd identity provider
+
+Configure an HTPasswd identity provider to create a single, static user with cluster administration privileges. You can log in to your cluster as the user to troubleshoot issues.
+
+.Procedure
+
+. From the link:https://console.redhat.com/[OCM console], navigate to the *Clusters* page and select the cluster that you need to configure identity providers for.
```

Comment:
Please double-check this use of "OCM console." I see "{cloud-redhat-com}" and "console.redhat.com" a lot, "{cloud-redhat-com} (OCM)" less frequently, and one "OCM" by itself after the initialism was defined in the module.

Reply:
@kalexand-rh thank you for your review and for highlighting this.

I have updated the line in this module and the other similar modules included in this assembly to use the `{cloud-redhat-com}` attribute instead. That attribute links to https://cloud.redhat.com/openshift, which now redirects to https://console.redhat.com/openshift. So, although the attribute directs the reader to the correct page ultimately, I think we need to discuss what to do with it in the broader sense as a team. For instance, should we just update the URL in the attribute, or change the attribute throughout the library to `:console-redhat-com:` also? I will add this to our team meeting agenda.

Reply:
Thank you, @pneedle-rh! Someone might need to volunteer to do a larger scrub to standardize our usage, and it'll be good to talk about it on the team meeting.